### DISCUSSION TITLE
H265 Support

### DISCUSSION THREAD

**Discussion by yusufziyacs** [webrtc-rs/webrtc#140] [Q&A]
Hello there. 
I've just started using webrtc-rs in order to convert RTSP streams into webrtc. I can easily create connections for H264 mime type. But I need also H265 tacks to be able to add. I see a codec for H265 but as far as I can see there isn't any Payloader implementation for it and when I add tracks with mime type "video/H265" i receive an error "ErrNoPayloaderForCodec". 
So how can I use this H265 codec or is there any other way to achieve to serve H265?

**Comment by rainliu** [comment]
I just created an issue to track this support
